src/app/helpers.js

```diff
@@ -63,9 +64,19 @@ function truncateLength(text, length = 100) {
   return truncate(text, { length, separator: /,? +/, ellipsis: '…' });
 }
 
+function capitalize(text) {
```

Comment:
Assuming the function `pageTitle` is no longer needed, `capitalize` should no longer be needed too.